<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSGArchive</class>
 <widget class="QMainWindow" name="SSGArchive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSGArchive</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/hicolor/512x512/apps/ssg.png</normaloff>icons/hicolor/512x512/apps/ssg.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="submittersName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Name of Submitter:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="resourceType">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>115</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>104</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Resource Type:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>590</y>
      <width>99</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QComboBox" name="domain">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>145</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>57</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Domain:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="contentClassification">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>285</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>153</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Content Classification:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="submissionDate">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="date">
     <date>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::YearSection</enum>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="vernacularTitle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>80</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>115</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Title in Vernacular:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="englishTitle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>80</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>145</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Title in English:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="authors">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>175</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Authors:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDateEdit" name="completionDate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="date">
     <date>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::YearSection</enum>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>225</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Date Completed:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="edition">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>255</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Edition:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sourceState">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>285</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Source State:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pdfState">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>315</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;PDF State:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="writingLanguage">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>225</y>
      <width>143</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>143</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Language of Writing:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="vernacularLanguage">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>250</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>255</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>147</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Vernacular Language:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="teachersGuide">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>315</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Teacher's Guide:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="publicationPlace">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>375</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Place of Publication:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="publisher">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>345</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Publisher:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>405</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Topic:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="primaryComments">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="outdates">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>361</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>475</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Documents outdated by this Document:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>581</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>The following information will allow us to see how this document is related to others.</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="secondaryComments">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>550</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Secondary Comments:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open Existing AIF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Unlike the above fields, the following information will not appear in the catalogue, but will be stored with the document in the archive and given to all users. Say anything you think users might find useful.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Primary Comments:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>65</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Submission Date:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>A .csv file created by this form is to be submitted with every request to archive a document.  The current instructions are available in &quot;HowToArchive yymmdd.doc&quot;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="topic">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andika Compact</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>345</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>111</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Pages:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pages">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <zorder>submittersName</zorder>
   <zorder>label_21</zorder>
   <zorder>resourceType</zorder>
   <zorder>label_22</zorder>
   <zorder>saveButton</zorder>
   <zorder>domain</zorder>
   <zorder>label</zorder>
   <zorder>contentClassification</zorder>
   <zorder>label_2</zorder>
   <zorder>submissionDate</zorder>
   <zorder>vernacularTitle</zorder>
   <zorder>label_4</zorder>
   <zorder>englishTitle</zorder>
   <zorder>label_5</zorder>
   <zorder>authors</zorder>
   <zorder>label_6</zorder>
   <zorder>line</zorder>
   <zorder>completionDate</zorder>
   <zorder>label_7</zorder>
   <zorder>line_2</zorder>
   <zorder>edition</zorder>
   <zorder>label_8</zorder>
   <zorder>sourceState</zorder>
   <zorder>label_9</zorder>
   <zorder>pdfState</zorder>
   <zorder>label_10</zorder>
   <zorder>writingLanguage</zorder>
   <zorder>label_11</zorder>
   <zorder>vernacularLanguage</zorder>
   <zorder>label_12</zorder>
   <zorder>teachersGuide</zorder>
   <zorder>label_13</zorder>
   <zorder>publicationPlace</zorder>
   <zorder>label_14</zorder>
   <zorder>publisher</zorder>
   <zorder>label_15</zorder>
   <zorder>label_16</zorder>
   <zorder>outdates</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
   <zorder>line_3</zorder>
   <zorder>secondaryComments</zorder>
   <zorder>label_20</zorder>
   <zorder>line_4</zorder>
   <zorder>primaryComments</zorder>
   <zorder>openButton</zorder>
   <zorder>label_23</zorder>
   <zorder>label_17</zorder>
   <zorder>label_25</zorder>
   <zorder>label_26</zorder>
   <zorder>topic</zorder>
   <zorder>label_24</zorder>
   <zorder>pages</zorder>
  </widget>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>openButton</tabstop>
  <tabstop>submittersName</tabstop>
  <tabstop>submissionDate</tabstop>
  <tabstop>vernacularTitle</tabstop>
  <tabstop>englishTitle</tabstop>
  <tabstop>authors</tabstop>
  <tabstop>resourceType</tabstop>
  <tabstop>domain</tabstop>
  <tabstop>completionDate</tabstop>
  <tabstop>edition</tabstop>
  <tabstop>sourceState</tabstop>
  <tabstop>pdfState</tabstop>
  <tabstop>publisher</tabstop>
  <tabstop>publicationPlace</tabstop>
  <tabstop>topic</tabstop>
  <tabstop>writingLanguage</tabstop>
  <tabstop>vernacularLanguage</tabstop>
  <tabstop>contentClassification</tabstop>
  <tabstop>teachersGuide</tabstop>
  <tabstop>pages</tabstop>
  <tabstop>primaryComments</tabstop>
  <tabstop>outdates</tabstop>
  <tabstop>secondaryComments</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
